packages/next/client/image.tsx

```diff
@@ -154,7 +162,7 @@ function generateImgAttrs({
     sizes = '100vw'
   }
 
-  src = callLoader({ src, quality, width: widths[last] })
+  src = callLoader({ src, quality, width: widths[last] }, loader)
 
   return { src, sizes, srcSet }
```

Comment:
this is incredibly nit-picky (and not related to this PR at all) but I refactored this return [to this](https://github.com/vercel/next.js/pull/20213/files#diff-ed7e3de801c4bc992b2796b6dd476f27320efef5d2911101e2ccb7b27c35866aR139-R150) and was feeling it 😅

Reply:
Cool! I added that refactor to this PR.